syncmaster/db/models/enums.py

```diff
@@ -0,0 +1,65 @@
+# SPDX-FileCopyrightText: 2023-2024 MTS PJSC
+# SPDX-License-Identifier: Apache-2.0
+from __future__ import annotations
+
+import enum
+
+
+class Status(enum.StrEnum):
```

Comment:
IMHO these enums could be moved to a file containing related model definition

Reply:
fixed in: https://github.com/MobileTeleSystems/syncmaster/pull/115